go.mod

```diff
@@ -1,37 +1,36 @@
 module github.com/asecurityteam/transportd
 
-go 1.19
+go 1.21
 
 require (
 	bitbucket.org/atlassian/go-asap v0.0.0-20190921160616-bb88d6193af9
-	github.com/SermoDigital/jose v0.9.2-0.20161205224733-f6df55f235c2
+	github.com/SermoDigital/jose v0.9.2-0.20180104203859-803625baeddc
```

Comment:
Are they actually bumping the code w/o bumping the version??

Reply:
I assume so :(